content/_data/changelogs/lts.yml

```diff
@@ -11239,6 +11239,48 @@
     message: |-
       Consistently notify job listeners when the job definition is updated from the REST API or command line interface.
 
+- version: "2.452.3"
+  date: 2024-07-10
+  changes:
+  - type: major bug
+    category: regression
+    pull: 9370
+    issue: 73246
+    authors:
+    - BobDu
+    message: |-
+      Show help text in the correct locale even if user has an alternate language option defined in their browser (regression in 2.444).
+  - type: major bug
+    category: regression
+    pull: 9409
+    issue: 73301
+    authors:
+    - basil
+    message: |-
+      Correctly highlight alerts (regression in 2.452.2).
+  - type: bug
+    category: bug
+    pull: 9347
+    issue: 73243
+    authors:
+    - mawinter69
+    message: |-
+      Show correct build history panel even when a pending parameterized build includes '$' in a parameter value.
```

Comment:
That a pending job with a parameter with `$`  lead to the an error was just one manifestation of the bug. The example with the `addInfoBadge`  step in the Jira causes this error to appear in the build history widget as well independent of a pending job. And potentially any other place where a symbol is rendered with a tooltip containing a `$` could cause problems, we might have just not encountered it yet. And probably there are other strings used in tooltips that could have caused problems.

Reply:
Thanks @mawinter69.  Would iy better describe the breadth of the fix if the first line of the description message said:

```
Show correct build history panel even when a $' character is included in the build history data.
```

Reply:
Maybe
```
Show correct build history panel even when a '$' character is included in a tooltip of build history data.
```

Reply:
Thanks!  Applied as 655053b3cd36314c6d3dffbe0ef456847483fe93